### DISCUSSION TITLE
みんなの自動翻訳のグループ登録

### DISCUSSION THREAD

**Discussion by noborus** [pgsql-jp/jpug-doc#2484] [General]
https://github.com/noborus/jpug-doc-tool/blob/main/MT.md を参考に
[みんなの自動翻訳＠TexTra®](https://mt-auto-minhon-mlt.ucri.jgn-x.jp/) でアカウントを作成したら、
ユーザーIDを書いてください。グループ登録することで、jpug向けのカスタマイズデータが使用できるようになります。

グループ管理者: noborus からグループ招待を送ります。

グループ登録が完了したら、 [https://github.com/noborus/jpug-doc-tool#機械翻訳](https://github.com/noborus/jpug-doc-tool#%E6%A9%9F%E6%A2%B0%E7%BF%BB%E8%A8%B3) を参考に設定ファイルを編集してください。
```console
$ jpug-doc-tool mt "This is a pen."
Using config file: /home/noborus/.jpug-doc-tool.yaml
c-1640_en_ja: これはペンです。
generalNT_en_ja: これはペンです。
```

のようにc-1640_en_jaの翻訳が使えるようになります。

**Comment by KenichiroTanaka** [comment]
ユーザ名ketanakaで登録しましたのでグループに入れていただけないでしょうか。

**Reply by noborus** [reply]
招待を送りました。

**Reply by KenichiroTanaka** [reply]
ありがとうございます。「参加」ボタンを押しました。

**Comment by koizumistr** [comment]
koizumistr でアカウント作りました。

**Reply by noborus** [reply]
招待を送りました

**Comment by noborus** [comment]
使用できるようになるのは公開情報なので、翻訳に参加しない人でもグループに参加できるようにしたいと思います。
使ってみたい人がいたら、お気軽にアカウントを作成して、IDを書いてください。

**Comment by tatsuo-ishii** [comment]
よくわかっていないので質問ですが、斉藤さん以外の人もこの機械翻訳システムを利用できるようにする目的（あるいはメリット）は何でしょうか？
現状、斉藤さんが機械翻訳＋斉藤さんのツールで下訳を用意してくださっているのですが（それは非常にありがたいです）、他の人も下訳を作れるようにするとかでしょうか？

**Reply by noborus** [reply]
はい。私以外の人ができるようになっていた方が良いのが一つと、`jpug-doc-tool mt "This is a pen."`のように文を渡して翻訳できるので、類似文として翻訳が入っていない箇所の翻訳やレビュー等にも利用できます。
入力フォームで翻訳すると一文ずつ複数のエンジンの翻訳が使えたりするので、より詳しく見ながら確認できます。
「みんなの自動翻訳」はカスタマイズとして、用語集や対訳集で学習させらるので共有して編集していくことで精度を上げることができます。

**Comment by tatsuo-ishii** [comment]
このサービスの利用規約を見ると、以下のように書いてあります。

> ５　開発会社等は、開発会社等アプリにおいては「本サービスを利用した結果として表示されるテキスト・画像等の配置される画面の全て」に、「Powered by NICT」と表示することに同意するものとします。

これに対応するために、日本語マニュアルに「Powered by NICT」と表示することになるのでしょうか？

**Reply by noborus** [reply]
日本語マニュアルに表示は必要ではないと解釈しています。

「みんなの自動翻訳」を利用したアプリケーションの結果が表示される画面に必要と読めるので、jpug-doc-toolの方で必要かと考えていたのですが、
CLIなのでどうしようかと考えてます。一応「みんなの自動翻訳」のサイトから便利アプリのOSSのリンクがあり、その中で https://github.com/ideuchi/trans
が紹介されており、これもCLIで結果を表示しているので、とりあえず問題ないかと思ってます。どこかに入れたほうがよければ入れる予定です。

翻訳結果をオープンソースへ使用する場合は、元のオープンソースのライセンスが適用されることで問題ないということが、
「みんなの自動翻訳」の質問・要望のところで回答されています。
規約とは別のところで、謝辞を入れるかどうかはJPUGでも検討されるという話でした。

**Reply by tatsuo-ishii** [reply]
念の為にNICTに聞いてみたのですが、クレジット表示は必要とのことです。ただ、全ページに表示する必要はなく、どこか一箇所で良いようです。個別コントリビューターへの謝辞のところにNICTへの謝辞を追加することになるでしょうか。
https://mt-auto-minhon-mlt.ucri.jgn-x.jp/content/qa/detail/?q_qa_id=1980

**Reply by KenichiroTanaka** [reply]
石井さん
NICTへの確認ありがとうございます。
NICTへの謝辞の追加方法を検討します。

**Reply by noborus** [reply]
> 念の為にNICTに聞いてみたのですが、クレジット表示は必要とのことです。ただ、全ページに表示する必要はなく、どこか一箇所で良いようです。個別コントリビューターへの謝辞のところにNICTへの謝辞を追加することになるでしょうか。 https://mt-auto-minhon-mlt.ucri.jgn-x.jp/content/qa/detail/?q_qa_id=1980

すみません。ありがとうございます。
